packages/server/lib/modes/run.js

```diff
@@ -880,7 +900,7 @@ module.exports = {
 
       // bar.tickTotal(float)
 
-      return videoCapture.process(name, cname, videoCompression, onProgress)
+      return videoCapture.process(name, cname, videoCompression, ffmpegChaptersConfig, onProgress)
```

Comment:
`ffmpegChaptersConfig` could just be a list of the tests with duration info and `title`, then `videoCapture` could handle the logic currently in `generateFfmpegChaptersConfig`